packages/backend-modules/access/lib/grants.js

```diff
@@ -85,6 +85,35 @@ const grantPerks = async (grant, recipient, campaign, t, pgdb, redis, mail) =>
     { concurrency: 1 },
   )
 
+const revokePerks = async (grant, recipient, campaign, pgdb) =>
+  Promise.map(
```

Comment:
To quote grandmother: "Forgive them for they know not what they do!"

I since discovered, instead of `Promise.map(…, { concurrency: 1 })`, there is a readable shorthand: [`Promise.mapSeries(…)`](http://bluebirdjs.com/docs/api/promise.mapseries.html).

Would opt-in to use that shorthand here, and in `grantPerks`, too.

```suggestion
  Promise.mapSeries(
```

(and drop `{ concurreny: 1 }` at end of declaration)